source/common/grpc/google_grpc_utils.cc

```diff
@@ -127,7 +127,7 @@ GoogleGrpcUtils::channelArgsFromConfig(const envoy::config::core::v3::GrpcServic
       break;
     }
     default:
-      NOT_REACHED_GCOVR_EXCL_LINE;
+      throw EnvoyException("Oneof name not set");
```

Comment:
The message here is accurate but it could be more helpful. Can you make it mention that it is related to gRPC config and channel args?

Reply:
Good point - changed fix to use protoc-gen-validate in latest commit.